tensorflow/c/c_api.h

```diff
@@ -900,6 +900,11 @@ TF_CAPI_EXPORT extern void TF_DeleteImportGraphDefOptions(
 TF_CAPI_EXPORT extern void TF_ImportGraphDefOptionsSetPrefix(
     TF_ImportGraphDefOptions* opts, const char* prefix);
 
+// Set default execution device for the nodes in the `graph_def` that will be imported into `graph`.
```

Comment:
Mention that this only overrides the device if one was not explicitly specified for the operation in the graph def?

Reply:
Updated